File: Python/beginner/1153.py
# https://judge.beecrowd.com/pt/problems/view/1153
# Fatorial Simples

# Requesting
usr_input = int(input())

# Main

if usr_input == 0:
    print(1)
else:
    fatorial = 1
    for i in range(1, usr_input + 1):
        fatorial *= i
    print(fatorial)
    
# By Fernando Serra
# https://github.com/fernandosserra

File: C/beginner/1153.c
// https://judge.beecrowd.com/pt/problems/view/1153
// Fatorial Simples

// Libraries
#include <stdio.h>

// Main
int main(){
    int usr_input, control1, fatorial;    

    scanf("%d", &usr_input);
    fatorial = 1;
    control1 = usr_input;

    while(control1 > 1){
        fatorial = fatorial * control1;
        control1--;
    }
    
    printf("%d\n", fatorial);

    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra